<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxAxis">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Invert Axis</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutInvert">
     <item>
      <widget class="QCheckBox" name="chkInvZ">
       <property name="text">
        <string>Z Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkInvY">
       <property name="text">
        <string>Y Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkInvX">
       <property name="text">
        <string>X Axis</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="labelWaitForResp">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seconds to Wait for Response</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinResponseWaitSec">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelZJogRate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-Jog Rate (inches or mm/min)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxUseMmManualCmds">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>431</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Use millimeters when sending manual commands (v0.8c+ only)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupZRateLimit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Z Rate Limiting</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutRateLimit">
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkLimitZRate">
       <property name="text">
        <string>Limit Z Rate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinZRateLimit">
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelZRateLimit">
       <property name="text">
        <string>Z-Rate Limit (inches or mm/min)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelXYRate">
       <property name="text">
        <string>XY Rate (inches or mm/min)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinXYRate">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
       <property name="value">
        <double>2000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinZJogRate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>260.000000000000000</double>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutExtras">
    <item>
     <widget class="QCheckBox" name="chkAggressivePreload">
      <property name="text">
       <string>Use aggressive preload strategy for Grbl</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxEnableDebugLog">
      <property name="text">
       <string>Write debugging log to home folder (requires restart)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>groupZRateLimit</zorder>
  <zorder>buttonBox</zorder>
  <zorder>groupBoxAxis</zorder>
  <zorder>labelWaitForResp</zorder>
  <zorder>spinResponseWaitSec</zorder>
  <zorder>labelZJogRate</zorder>
  <zorder>checkBoxUseMmManualCmds</zorder>
  <zorder>doubleSpinZJogRate</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
